packages/repository/src/repository.ts

```diff
@@ -166,10 +166,10 @@ export class Repository<
     return deserialize(document) as TEntity;
   }
 
-  async find(filter: FilterQuery<TEntity>): Promise<TEntity[]> {
+  async find(filter: FilterQuery<TEntity>, options?: QueryOptions): Promise<TEntity[]> {
```

Comment:
what options are you planning to use? we don't want our API to be too coupled with `mongoose`. we are planning to use the same API for other databases such as `SQLite`